### PR TITLE
My week 1 solution

### PR DESCRIPTION
**Pull request by kunmi02**
## Description

- I solved the Week 1 task.
- I created a new file inside submissions/kunmi02/Week1solution/
- I name this file fizzbuzz.java, hence we have the fizzbuzz solution in submissions/kunmi02/Week1solution/fizzbuzz.java

### CODE REVIEW

**Comment by ibironke-yekinni** [comment]
> Well done @kunmi02

File: submissions/kunmi02/week1solution/fizzbuzz.java [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+            } // end of fizzbuzz
+            else if ((i % 3)==0){ //condition fizz
+                System.out.println ("Fizz" + "\n");
+            } //end of fizz
+            else if ((i % 5)==0){ // condition for buzz
+                System.out.println ("Buzz" + "\n");
+            } // eond of buzz
+            else{
+            System.out.println ( i + "\n"); //print the number if it does not satisfy the fizz,buzz or fizzbuzz condition
+        }
```
Comment (codedentwickler) [inline submissions/kunmi02/week1solution/fizzbuzz.java]: Indentation the closing brace `}` for the else block seems off a bit.

File: submissions/kunmi02/week1solution/fizzbuzz.java [hunk 2]
```diff
@@ -0,0 +1,25 @@
+
+public class Fizzbuzz {
+
```
Comment (codedentwickler) [inline submissions/kunmi02/week1solution/fizzbuzz.java]: Too much lines spaces, a single line might be enough. What do you think?

File: submissions/kunmi02/week1solution/fizzbuzz.java [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+    
+    public static void main(String[] args) {
+        // TODO code application logic here
+        int endcount=100; //end of my counter
+        for (int i=1;i<=endcount;i++){ //initialized a for loop to count from 1 - 100
+            
+            if ((i % 3)==0 && (i % 5)==0){ // condition for fizzbuzz
+                System.out.println ("FizzBuzz" + "\n");
+            } // end of fizzbuzz
```
Comment (codedentwickler) [inline submissions/kunmi02/week1solution/fizzbuzz.java]: no need to add this comment `end of fizzbuzz`, seems redundant. it's all over the place.

File: submissions/kunmi02/aboutme.md [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+
+ABOUT CODING
+
+I write java very well, I love java especially for mobile development. I write some php/css too. i dont like javascript at all because of the stress that comes with it (you just can get the errors on your codes except through proper viewing of codes line by line)
+
+I dont have any favourites emoji. I love challenges and also working on new things. I like multi tasking myself too.
+
+You can reach out to me on the following platforms:
+
+    twitter.com/ibrogodo
```
Comment (codedentwickler) [inline submissions/kunmi02/aboutme.md:14]: You can include a link to github profile here.

File: submissions/kunmi02/week1solution/fizzbuzz.java [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+            else if ((i % 3)==0){ //condition fizz
+                System.out.println ("Fizz" + "\n");
+            } //end of fizz
+            else if ((i % 5)==0){ // condition for buzz
+                System.out.println ("Buzz" + "\n");
+            } // eond of buzz
+            else{
+            System.out.println ( i + "\n"); //print the number if it does not satisfy the fizz,buzz or fizzbuzz condition
```
Comment (codedentwickler) [inline submissions/kunmi02/week1solution/fizzbuzz.java]: No need to add `\n`, you used println() already.

File: submissions/kunmi02/aboutme.md [hunk 6]
```diff
@@ -0,0 +1,15 @@
+Brief Introduction
+
```
Comment (codedentwickler) [inline submissions/kunmi02/aboutme.md:2]: Nice Job. Just make changes suggested and we can merge this.

File: submissions/kunmi02/aboutme.md [hunk 7]
```diff
@@ -0,0 +1,15 @@
...
+My name is Azeez Ibrahim Akinkunmi from University of Ibadan. I am computer science Student.
+
+
+ABOUT CODING
+
+I write java very well, I love java especially for mobile development. I write some php/css too. i dont like javascript at all because of the stress that comes with it (you just can get the errors on your codes except through proper viewing of codes line by line)
+
+I dont have any favourites emoji. I love challenges and also working on new things. I like multi tasking myself too.
```
Comment (codedentwickler) [inline submissions/kunmi02/aboutme.md:10]: Really dude, try and get one 😄.

**Review by codedentwickler — CHANGES_REQUESTED** [review]
Great job.

**Comment by codedentwickler** [comment]
@kunmi02 noticed you updated your submission but you didn't attend to all the comments. You can remove the redundant comments in the code.

**Comment by kunmi02** [comment]
Am still working on the file.
Am yet to push the changes in my code.
Will do that soon.

**Comment by kunmi02** [comment]
I have made  all the suggested changes to the files.

**Comment by kunmi02** [comment]
@codedentwickler please go through my changes.

File: submissions/kunmi02/week1solution/fizzbuzz.java [hunk 8]
```diff
@@ -0,0 +1,22 @@
+public class Fizzbuzz {    
+    public static void main(String[] args) {
+        // TODO code application logic here
+        int endcount=100; //end of my counter
+        for (int i=1;i<=endcount;i++){ //initialized a for loop to count from 1 - 100
+            
+            if ((i % 3)==0 && (i % 5)==0){ // condition for fizzbuzz
+                System.out.println ("FizzBuzz" );
```
Comment (codedentwickler) [inline submissions/kunmi02/week1solution/fizzbuzz.java]: This is a method call, it be `out.println();` not `out.println ();` Can you verify this formatting issue

**Review by codedentwickler — CHANGES_REQUESTED** [review]

File: submissions/kunmi02/week1solution/fizzbuzz.java (hunk 8 above)
Comment (kunmi02) [reply submissions/kunmi02/week1solution/fizzbuzz.java]: @codedentwickler Boss!! have rectified the formatting issue. please go through sir!!

**Review by codedentwickler — APPROVED** [review]

**Pull request merged**